features/account/customer_account/address_book/adding_address.feature

```diff
@@ -14,4 +14,4 @@ Feature: Adding a new address to the book
         And I specify the address as "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States", "Arkansas"
         And I add it
         Then I should be notified that the address has been successfully added
-        And the address assigned to "Lucifer Morningstar" should appear in my book
```

Comment:
I replaced this step because the previous one only tested addresses in `'#sylius-addresses'` selector

Reply:
I mean, that we could leave this one and just add the second one

Reply:
I don't really get it. 
Because : 
- `@Then this address should be assigned to :fullName` call 
-> `AddressBookContext::thisAddressShouldHavePersonFirstNameAndLastName()` that calls
-> `AddressBook\IndexPage::hasAddressOf($fullname)` that calls
->  `AddressBook\IndexPage::getAddressOf($fullname)` that uses element `addresses` using `#sylius-addresses` as selector which doesn't wrap the default address.

Reply:
Ok, totally forgot that address is not only _marked_ as default, but there is a separate *default address* section and then other addresses. Then it's fine :) 👍